src/mechanics/mesh/UnitMeshFem.h

```diff
@@ -6,6 +6,11 @@ namespace NuTo
 namespace UnitMeshFem
 {
 
+//! @brief creates a 1 dimensional mesh from (0) -- (1) with numX divisions
+//! @param numX number of divisions in x direction
+//! @return created mesh
+MeshFem CreateTrusses(int numX);
```

Comment:
I would really like to see a simple test every new method. Have a look at the [existing ones](https://github.com/nutofem/nuto/blob/PDE_reviewed/test/mechanics/mesh/UnitMeshFem.cpp). Nothing fancy. You could create a mesh out of 2 elements and check if `numNodes == 3` and if they are nodes at `0, 0.5 and 1`. (no exception of `mesh.NodeAtCoordinate(0.5)`. IMO that is enough.

Reply:
Was already on my list, just give me some time xD

Reply:
Ideally, you write the test first :stuck_out_tongue:

Reply:
What do you think about a rename? "Trusses" already suggests mechanics again, so I'd prefer something with "line" or "interval" instead.